### PR TITLE
[10.0][MIG] medical pharmacy us forward port

### PR DESCRIPTION
**Pull request by woodbrettm**
* Extracted from #180 

### Status
Not ready for official review, neither am I 100% sure to go ahead with as I haven't performed a diff audit. @LasLabs/admins same goes here, I think this still needs to be migrated?

Thanks,
Brett

### CODE REVIEW

File: medical_pharmacy_us/models/medical_pharmacy.py [hunk 1]
```diff
@@ -1,43 +1,26 @@
...
-        string='NAPB #',
-        comodel_name='res.partner.id_number',
-        compute=lambda s: s._compute_identification(
-            'napb_num', 'NAPB',
-        ),
-        inverse=lambda s: s._inverse_identification(
-            'napb_num', 'NAPB',
-        ),
-        help='National Boards of Pharmacy Id # - 7 digits',
+    nabp_num = fields.Char(
```
Comment (tedsalmon) [inline medical_pharmacy_us/models/medical_pharmacy.py]: `napb_num` not `nabp_num`

**Review by tedsalmon — COMMENTED** [review]
I do believe this still needs forward porting, but I'm not sure if @lasley has different plans for it.

**Comment by lasley** [comment]
Why would we not be continuing forward?

**Comment by lasley** [comment]
@obulkin  please fix this PR. We just need tests to pass

**Comment by obulkin** [comment]
This PR has been superseded by changes to ``release/10.0`` that were made elsewhere and is no longer needed so I'm closing it.

**Pull request closed without merging**